### ISSUE TITLE
Issue about the NT_Xent loss

### ISSUE THREAD

**Issue by 980202006** [Spijkervet/SimCLR#43]
In the forward function, logits is the splicing of the constructed positive samples and negative samples, but when calculating the loss, both positive samples and negative samples are given label 0. Is there a problem? The original paper implementation uses one-hot, that is, the positive sample should be 1.

**Comment by 980202006** [comment]
I get it, sorry the implementation is indeed right.
